crypto/fipsmodule/ec/ec.c

```diff
@@ -662,7 +733,25 @@ const char *EC_curve_nid2nist(int nid) {
   return NULL;
 }
 
+const char *EC_curve_nid2name(int nid) {
```

Comment:
`EC_curve_nid2nist` and `EC_curve_nist2nid` are public functions I guess?

Reply:
correct